packages/dataviews/src/dataviews.tsx

```diff
@@ -134,6 +135,12 @@ export default function DataViews< Item >( {
 						setOpenedFilter={ setOpenedFilter }
 					/>
 				</HStack>
+				{ view.type === LAYOUT_GRID && (
```

Comment:
Instead of putting this in its own menu, I think we should put it in the ViewActions instead and maybe offer a few different aspect ratio choices?